src/pages/home/Home.tsx

```
@@ -23,35 +23,82 @@ const Home = () => {
+        threshold: [0.3, 0.5, 0.7],
       }
     );
 
-    observer.observe(heroSection);
-    observer.observe(applySection);
+    sections.forEach((section) => observer.observe(section));
 
-    return () => {
-      observer.disconnect();
-    };
+    return () => observer.disconnect();
```

Biome (2.3.13)

[error] This callback passed to forEach() iterable method should not return a value.

Either remove this return or remove the returned value.


(lint/suspicious/useIterableCallbackReturn)